catalog/sat-io/projects_sat-io_open-datasets_GLOBathy_GLOBathy_bathymetry.jsonnet

```diff
@@ -0,0 +1,101 @@
+local id = 'projects/sat-io/open-datasets/GLOBathy/GLOBathy_bathymetry';
+local subdir = 'sat-io';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+
+local license = spdx.cc0_1_0;
+
+local basename = std.strReplace(id, '/', '_');
+local base_filename = basename + '.json';
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+{
+  // TODO: remove 'gee:skip_indexing'
+  'gee:skip_indexing': true,
+  'gee:user_uploaded': true,
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+  ],
+  id: id,
+  title: 'GLOBathy Global lakes bathymetry dataset',
+  'gee:type': ee_const.gee_type.image_collection,
+  description: |||
+    The GLObal Bathymetric (GLOBathy) dataset, comprising data on over 1.4 million waterbodies globally, has been meticulously developed to harmonize with the widely recognized HydroLAKES dataset. Utilizing a sophisticated Geographic Information System (GIS)-based framework, GLOBathy constructs detailed bathymetric maps by integrating maximum depth estimates and geometric/geophysical attributes sourced from HydroLAKES. Ensuring data accuracy and reliability, GLOBathy undergoes stringent validation procedures involving 1,503 waterbodies and a diverse range of observed data sources. Consequently, GLOBathy stands as a robust and comprehensive dataset for hydrography and aquatic sciences, offering invaluable resources for researchers and professionals in these fields.
+  |||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id),
+  keywords: [
+    'bathymetry',
+    'lake',
+    'hydrology',
+  ],
+  providers: [
+    ee.producer_provider('Bahram Khazaei', 'https://springernature.figshare.com/articles/dataset/GLOBathy_Bathymetry_Rasters/13404635'),
+    ee.host_provider(self_ee_catalog_url),
+  ],
+  extent: ee.extent_global('null', 'null'),
```

Comment:
The extent requires a start date; end date is optional if the dataset is periodically updated. Doesn't have to be super-precise; it can be Jan 1 of the first year in the set, for example.